artemis-server/src/main/java/org/apache/activemq/artemis/core/paging/impl/Page.java

```diff
@@ -223,7 +224,7 @@ private int read(StorageManager storage, ActiveMQBuffer fileBuffer, List<PagedMe
 
    public synchronized void write(final PagedMessage message) throws Exception {
       if (!file.isOpen()) {
-         return;
+         throw new ActiveMQIOErrorException("can't write to closed file " + file);
```

Comment:
Use a code on loggers?

Reply:
ok. I thought about a jira before but I didn't do it as the code change is so few. :)